src/styles/utils/_function.scss

```diff
@@ -0,0 +1,3 @@
+@function getWidth ($width,$count,$shift) {
```

Comment:
It's recommended to use more descriptive names for the function and its parameters. Consider renaming the function to 'calculateTotalWidth' and the parameters to 'columnWidth', 'numberOfColumns', and 'gapWidth'.